src/Workers/Worker.ts

```diff
@@ -1 +1,59 @@
-// nothing yet...
\ No newline at end of file
+import cv from '../../build/opencv_js'
+const ctx: Worker = self as any;
+
+ctx.onmessage = (e: MessageEvent<any>) => {
+    const msg = e.data;
+    switch (msg.type) {
+        case "loadTrackables": {
+            loadTrackables(msg);
+            return;
+        }
+    }
+};
+
+const ValidPointTotal = 15
+
+const BlurSize = 4
+
+var template_keypoints_vector;
+
+var template_descriptors;
+
+var homography_transform;
+
+var corners: Array<object> = [];
+
+var opencv;
+
+const loadTrackables = async (msg: any) => {
+    opencv = await cv();
```

Comment:
maybe we can init opencv outside **loadTrackables**?

Reply:
done with https://github.com/webarkit/webarkit-cv/pull/5/commits/73af4b83b6ccc710a267e7af2e03164c8a8912da